robot_tests/robot_tests/test_executor.py

```diff
@@ -0,0 +1,13 @@
+import os
+from robot import run
+package_directory = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestExecutor:
+    def __init__(self, host: str, port: int):
+        self.host = host
+        self.port = port
+
+    def execute_tests(self, include_tags=[], exclude_tags=[], output_dir="robot_results"):
```

Comment:
The output should be in the UTF workspace directory; see the `upgrade_testing_framework.core.workspace_wrangler` for more details about that.

Reply:
Cool needed a better place to put this, will add in